<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>426</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Масштабирование:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Масштабировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Вращение:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Вращать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Перемещение:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Переместить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Обзор...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
